_events/15.md

```diff
@@ -0,0 +1,36 @@
+---
+title:  "Learning Group Session #4 📚"
+date:   2020-01-16 19:00:00 +0100
+tags:
+ - learning
+---
+
+## Tech Workers Coalition Berlin - Learning Club
+Learning is central to workers' empowerment. Without knowledge on the complex world we operate in, we are prey to the interests of those who capitalize on exploitation. Tech Worker Coalition organizes regular sessions in the form of reading groups to read and discuss articles, books and pamphlets about a variety of topics that sit on the boundary between technology critique, politics, philosophy and sociology. The role of the Tech Worker, the possible strategies and the history of worker organizations are also targets to our work.
+
+## {{ page.date | date: '%A, %d %B %Y at %H:%M' }}
+For the first session of the year we want to continue the work that began in our previous sesssion and discuss Workers' Inquiry as a tool rooted in the present. Mind that participating in the first session is not critical to understand what we are gonna discuss this time. Last month the focus was on the history of the Inquiry, with a very lengthy and over-detailed reading that put off some of us with an excessive dose of labour theory. This time we want to do something different: we want to see how TWC is using this tool in the USA and how they adapted it to the current reality of the Tech Industry.
```

Comment:
session is misspelled the first time, and let's add the facebook link https://www.facebook.com/events/719129231910637/